frontend/global-styles/custom-alert.scss

```
@@ -0,0 +1,22 @@
+/*!
+ * SPDX-FileCopyrightText: 2023 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+.alert-hedgedoc:before{
+  content: ' ';
+  background:url('/icons/android-chrome-192x192.png');
```

formatting (run prettier?)

```suggestion
  background: url('/icons/android-chrome-192x192.png');
```